src/rest/rest_web_server.hpp

```diff
@@ -81,13 +81,16 @@ class RestWebServer : public MainloopProcessor
     void      UpdateConnections(const fd_set &aReadFdSet);
     void      CreateNewConnection(int32_t &aFd);
     otbrError Accept(int32_t aListenFd);
+    bool      ParseListenAddress(const std::string listenAddress, struct in6_addr *sin6_addr);
     void      InitializeListenFd(void);
     bool      SetFdNonblocking(int32_t fd);
 
     // Resource handler
     Resource mResource;
     // Struct for server configuration
     sockaddr_in6 mAddress;
+    // Listen address
+    const std::string mRestListenAddress;
```

Comment:
Can we reuse the `mAddress`?
For example, change the type of `mAddress` to `std::string` or initialize `mAddress` in the constructor with the configured REST address.